hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/web/TestWebHDFS.java

```diff
@@ -2230,6 +2230,25 @@ public void testLinkTarget() throws Exception {
     }
   }
 
+  @Test
+  public void testFileLinkStatus() throws Exception {
+    final Configuration conf = WebHdfsTestUtil.createConf();
+    cluster = new MiniDFSCluster.Builder(conf).numDataNodes(3).build();
```

Comment:
Use try with resources  & you don't need 3 datanodes, 1 datanode is enough and that is by default, you can drop `` .numDataNodes(3)``

Reply:
I'll make the required code changes promptly